src/Components/CreateImageWizardV2/steps/Oscap/OscapProfileInformation.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import {
+  Spinner,
+  TextContent,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+} from '@patternfly/react-core';
+
+import { useAppSelector } from '../../../../store/hooks';
+import { useGetOscapCustomizationsQuery } from '../../../../store/imageBuilderApi';
+import {
+  selectDistribution,
+  selectProfile,
+} from '../../../../store/wizardSlice';
+
+const OscapProfileInformation = (): JSX.Element => {
+  const release = useAppSelector((state) => selectDistribution(state));
+  const oscapProfile = useAppSelector((state) => selectProfile(state));
+
+  const {
+    data: oscapProfileInfo,
+    isFetching: isFetchingOscapProfileInfo,
+    isSuccess: isSuccessOscapProfileInfo,
+  } = useGetOscapCustomizationsQuery(
+    {
+      distribution: release,
+      // @ts-ignore
```

Comment:
Again - it would be good to be able to get rid of this. Let's address the rest of the comments and then we'll revisit this one.

Reply:
The thing is, we can't get rid of this `ts-ignore`, the linter doesn't understand that the query is not going to get executed if the profile is undefined, because of the syntax of the skip token. So my take on this is that we should keep the ignore comment but add some description to it so that we have a reminder on why it's there.

Reply:
Aha, yeah in cases where we truly can't get rid of the ignore I completely agree with adding a comment with a reason why it's there.